apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -101,24 +110,57 @@ export const saveStudent = (studentId) => {
   };
 };
 
-// Adds a student, with the given row id (studentId), from an addRow or
-// a newStudentRow.
-export const addStudent = (studentId) => {
+// Saves all RowType.STUDENT currently being edited and adds all
+// RowType.NEW_STUDENT currently being edited.
+export const saveAllStudents = () => {
   return (dispatch, getState) => {
     const state = getState().manageStudents;
-    dispatch(startSavingStudent(studentId));
-    addStudentOnServer(state.editingData[studentId], state.sectionId, (error, data) => {
+
+    // Currently, every update is an individual call to the server.
```

Comment:
Do we plan to change this later?

Reply:
Maybe for V2. I think it could be useful to think about changing the API once we create the "edit all" feature.